person.rb

```diff
@@ -20,6 +21,10 @@ def correct_name
     @name
   end
 
+  def add_rental(rental)
+    @rentals.push(rental)
+  end
```

Comment:
- [ ] @sanieni6 great work in adding the add_rental method, however for us to achieve the 'has-many' association between a person and book through the rental class, we need to pass ``date, book, self`` as  parameters when creating a new rental